tests/test_transliterate.py

```diff
@@ -152,3 +152,10 @@ def test_pronunciate(self):
         self.assertIsNotNone(pronunciate("มข.", engine="w2p"))
         self.assertIsNotNone(pronunciate("มช.", engine="w2p"))
         self.assertIsNotNone(pronunciate("jks", engine="w2p"))
+
+    def test_puan(self):
+        self.assertEqual(puan("นาริน"), "นิน-รา")
+        self.assertEqual(puan("นาริน", False), "นินรา")
+        self.assertEqual(puan("สวัสดี"), "สะ-หี-ดวัด")
```

Comment:
If politeness is a concern, we might change this input to something else.

Reply:
I fixed. Sorry, I don't check it.